README.md

```diff
@@ -9,10 +9,11 @@ MVP at https://www.paironauts.org
 To start the Paironauts Phoenix server:
 
   * Install dependencies with `mix deps.get`
+  * Create Ecto migrations directory using...
+    * (Mac/Linux) using `mkdir -p priv/repo/migrations`
+    * (Windows) TBD
   * Create and migrate your database with `mix ecto.create && mix ecto.migrate`
-  * Install Node.js dependencies with `cd assets && npm install`
+  * Install Node.js dependencies with `(cd assets && npm install)`
```

Comment:
What are these parens for?

Reply:
Keeps you in the same directory.

Reply:
more specifically, you'll be in the same directory after the command is finished running.

Reply:
Gotcha. Didn't know that. Learn something knew everyday.

With that understanding it seems to me this is a good change